src/_components/NavBarContainer.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import toggleVisibilityAction from '../_actions/navBarToggle.action';
+import { NavBarComponent } from '../components/NavBarComponent';
+
+const NavBarContainer = props => (
```

Comment:
`NavBarContainer` is a redundant component, as it is just rendering the `NavBarComponent`. Delete it as it is not needed.  Ideally, a container file should contain `mapStateToProps`, `mapDispatchToProps` and `connect`. You can also include any other helper function that maps the state to what you want. You don't have to worry about creating a container component such as `NavBarContainer` to connect to. Do the connection directly to `NavBarComponent`.  Have a look at this [redux example](https://redux.js.org/basics/exampletodolist#containers-filterlink-js) and read more about containers in this [redux explanation](https://redux.js.org/basics/usagewithreact#implementing-container-components) and [react-redux docs](https://react-redux.js.org/docs/introduction/quick-start).

Therefore, I'm expecting to only see the following in the file:
```jsx
const mapStateToProps = ({ navBarVisibility }) => {
  const { isVisible } = navBarVisibility;
  return {
    isVisible
  };
};

export default withRouter(connect(mapStateToProps, {
  toggleVisibilityAction 
})(NavBarComponent));
```